<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayListDialog</class>
 <widget class="QDialog" name="PlayListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>291</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>プレイリスト</string>
  </property>
  <layout class="QVBoxLayout" name="_verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="_horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="_buttonAdd">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>21</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_buttonRemove">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>21</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>_buttonAdd</tabstop>
  <tabstop>_buttonRemove</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
